### PR TITLE
Example Notebooks: Compute Updates for v2.1.1

### PR DESCRIPTION
**Pull request by stephencpope**
updates the Compute example notebooks for correctness and best practices for v2.1.1

### CODE REVIEW

File: guides/batch-compute/01 Hello World.ipynb [hunk 1]
```diff
@@ -100,7 +110,11 @@
...
     ")\n",
-    "async_func.save()"
+    "async_func.save()\n",
+    "\n",
+    "# wait for the function to be ready (will take a few minutes)\n",
+    "print(\"waiting for function to be ready\")\n",
+    "async_func.wait_for_completion()"
    ]
   },
   {
```
Comment (mat-savage) [inline guides/batch-compute/01 Hello World.ipynb:120]: [not a blocker] Sorry for the placement of this comment but I can't put it where I want - above, on line 60, it reads

> Next, we'll create a very basic `hello_world` function that imports a common geospatial Python package and returns a given argument.

Our hello world example does not import anything

File: guides/batch-compute/01 Hello World.ipynb [hunk 2]
```diff
@@ -136,15 +152,12 @@
...
-   "id": "a7a3fb3a-64b3-4b36-92ad-8b7305adb90f",
-   "metadata": {},
-   "source": [
-    "async_func.wait_for_completion()"
+    "\n",
+    "There are multiple ways to wait for completion.\n",
+    " * You can wait on an individual job as in the previous cell.\n",
+    " * You can wait for all pending and running jobs for the function to complete using `async_func.wait_for_completion()`.\n",
+    " * You can navigate to the Compute monitor app at [app.descarteslabs.com/monitor](https://app.descarteslabs.com/monitor).\n",
+    " * You can embed the same directly in your notebook as in the following cell."
```
Comment (mat-savage) [inline guides/batch-compute/01 Hello World.ipynb:160]: embedding the monitor is a cool idea!

**Review by mat-savage — APPROVED** [review]
Just had one non-blocking comment

File: guides/batch-compute/01 Hello World.ipynb (hunk 1 above)
Comment (stephencpope) [reply guides/batch-compute/01 Hello World.ipynb:120]: bah! That had originally been copied from the guide examples (in the docs). I'll fix it.

**Review by cmoultonatx — APPROVED** [review]

**Pull request merged**